src/main/resources/assets/tconstruct/book/encyclopedia/en_us/upgrades/armor/general/simple_absorption_absorption.json

```diff
@@ -0,0 +1,17 @@
+{
+  "modifier_id": "simple_absorption:absorption",
+  "text": [
+    {
+      "text": "Gives you half an absorption heart per level."
+    },
+    {
+      "text": "Absorption will regenerate over time and have over 9 shanks of hunger, and much faster when you are full hunger",
+      "paragraph": true
+    }
+  ],
+  "more_text_space": true,
+  "tool_filter": "tconstruct:modifiable/armor",
```

Comment:
Probably don't need the tool filters, they are only used when a modifier has multiple pages. For these might as well leave it out in case we change the recipe later.